_data/components/realInput.yml

```diff
@@ -30,4 +30,11 @@ attributes: [{
   description : "Maximum number of decimals required"
 },{
   name: "decimal-separator"
-}]
+},
+{
+  name: "strict",
+  type: "boolean",
+  default : "false",
+  required : "",
+  description : "Indicates whether the component should be strict when displaying errors"
```

Comment:
We should explain it better. It is important to remark that with strict mode the component only accepts the number of decimals configured into rendered view and data model.